Source/Examples/Tutorial.swift

```diff
@@ -3,8 +3,11 @@
 import Foundation
 import MLX
 
+/// Tutorial program.
+///
 /// mlx-swift tutorial based on:
-/// https://github.com/ml-explore/mlx/blob/main/examples/cpp/tutorial.cpp
+///
+/// - [](https://github.com/ml-explore/mlx/blob/main/examples/cpp/tutorial.cpp)
```

Comment:
What does that do?

Reply:
The link doesn't work in the first line (it is the title).  I like the look of it in a bullet list with the indent.